sympy/parsing/latex/_antlr/latexparser.py

```diff
@@ -15,10 +15,11 @@
 from io import StringIO
 import sys
 
+
 def serializedATN():
     with StringIO() as buf:
         buf.write(u"\3\u608b\ua72a\u8133\ub9ed\u417c\u3be7\u7786\u5964\3")
-        buf.write(u"K\u01b2\4\2\t\2\4\3\t\3\4\4\t\4\4\5\t\5\4\6\t\6\4\7\t")
+        buf.write(u"M\u01b2\4\2\t\2\4\3\t\3\4\4\t\4\4\5\t\5\4\6\t\6\4\7\t")
```

Comment:
Note: #19980 will make these changes collapsed by default in github.

Reply:
I hope this isn't a blocker for this PR.

Reply:
I don't think #11980 is particularly controversial. What might be a blocker to all these PRs is @asmeurer's opinion in https://github.com/sympy/sympy/pull/19980#issuecomment-675727311